confluent_kafka/__init__.py

```diff
@@ -634,3 +634,25 @@ def __repr__(self):
 
     def __str__(self):
         return "{}".format(self.partition)
+
+
+class ThrottleEvent (object):
+    """
+    ThrottleEvent contains details about a throttled request.
+
+    This class is typically not user instantiated.
+
+    :ivar broker_name str: The hostname of the broker which throttled the request
+    :ivar broker_id int: The broker id
+    :ivar throttle_time float: The amount of time the broker throttled (delayed) the request(seconds)
```

Comment:
I would probably do: "The amount of time (in seconds) the ..", or at least add a space before the paren: "request (seconds)"